### ISSUE TITLE
Break lambda into multiple lambdas

### ISSUE THREAD

**Issue by Bjorn248** [Bjorn248/aws_pricing_data_ingestor#14]
* 1 producer lambda that gets all services
* SQS that takes each service pricing CSV url
* consumer lambda that processes each pricing service CSV in parallel

**Comment by Bjorn248** [comment]
This has been completed in the migrate to planetscale branch...however more needs to be done, will track that in another issue.
